x-pack/solutions/observability/plugins/apm/public/components/app/alerts_overview/index.tsx

```diff
@@ -48,30 +48,45 @@ export function AlertsOverview() {
       alertsTableConfigurationRegistry,
     },
     notifications,
-    data: {
-      query: {
-        timefilter: { timefilter: timeFilterService },
-      },
-    },
+    data,
+    dataViews,
+    http,
+    spaces,
     uiSettings,
     observability: { observabilityRuleTypeRegistry },
   } = services;
+  const {
+    query: {
+      timefilter: { timefilter: timeFilterService },
+    },
+  } = data;
 
   const useToasts = () => notifications!.toasts;
 
-  const apmQueries = useMemo(() => {
+  const apmFilters: Filter[] = useMemo(() => {
```

Comment:
I changed Query to Filter in order to pass it to the Controls as well.

Reply:
I think you could omit the `Filter[]` here. The `useMemo` return type is already `Filter[]`